.env.dev.example

```diff
@@ -13,12 +13,8 @@ ETH_RPC_HOST=http://localhost
 ETH_RPC_PORT=8545
 ETH_WALLET_PK=0x06dd0990d19001c57eeea6d32e8fdeee40d3945962caf18c18c3930baa5a6ec9
 
-TYPEORM_CONNECTION = sqlite
-TYPEORM_HOST = localhost
-TYPEORM_USERNAME = root
-TYPEORM_PASSWORD = root
-TYPEORM_DATABASE = ./data/anchor.db.sq3
-TYPEORM_PORT = 5432
-TYPEORM_SYNCHRONIZE = true
-TYPEORM_LOGGING = true
-TYPEORM_ENTITIES = build/models/**/*.js
+DB_HOST = localhost
```

Comment:
Had a look at CircleCI workflow, I am just confused. We use `DATABASE_URL` on CircleCI but here we use full set of variables. Could you please tell why?

Reply:
In our tests we use "@databases/pg-test" which uses docker to launch a postgres db. This allows anyone to run all the tests locally without having to set up their own postgres. 

If DATABASE_URL is set, pg-test does nothing. So you can run tests locally using your own postgres instances by setting DATABASE_URL to point to it.

For use of pg-test with CircleCI, it was recommended via pg-test's docs to launch a postgres using CircleCI and then set DATABASE_URL so pg-test uses it.

>But, no, you could use single variable for connection string here, and pass it to the config file. That would be a breaking change, but it is possible still, and we are the only ones running CAS at the moment, so no one external is affected really IMO.

Yes that is true. I originally kept the connection as an object to minimize the typeorm --> knex changes BUT since all of our ENV vars are prefixed with TYPEORM_XXX we will have to make a bunch of changes anyway. I'll make this change.

Reply:
Oh, this is so very complicated. I am glad you own it.